ci/scripts/glib_test.sh

```diff
@@ -61,6 +61,14 @@ test_subproject() {
                --with-cppflags=-D_LIBCPP_DISABLE_AVAILABILITY
     fi
 
+    # Install consistent version of red-arrow for given arrow-glib
+    local -r arrow_glib_version=$(pkg-config --modversion arrow-glib | sed -e 's/-SNAPSHOT$//g' || :)
+    local red_arrow=""
```

Comment:
Sorry for not working `gem install red-arrow:X.Y.Z pkg/*.gem`...

We can remove the `red_arrow` variable entirely.